diesel/src/expression/helper_types.rs

```diff
@@ -8,6 +8,8 @@ pub type SqlTypeOf<Expr> = <Expr as Expression>::SqlType;
 pub type AsExpr<Item, TargetExpr> = AsExprOf<Item, SqlTypeOf<TargetExpr>>;
 pub type AsExprOf<Item, Type> = <Item as AsExpression<Type>>::Expression;
 
+#[macro_export]
```

Comment:
It was just missing the `#[macro_use]`

Do you want me to move the pg/expression/helper_types.rs in there ?